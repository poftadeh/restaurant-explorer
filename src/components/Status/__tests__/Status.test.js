import React from 'react';
import { shallow } from 'enzyme';
import { Status } from '../index';

it('renders <Status /> properly', () => {
  const wrapper = shallow(<Status />);
  expect(wrapper).toMatchSnapshot();
});
