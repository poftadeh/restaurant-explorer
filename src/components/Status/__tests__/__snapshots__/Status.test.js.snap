// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Status /> properly 1`] = `
<img
  alt="Plate of Food"
  className="status-image"
  src="food.svg"
/>
`;
